docs/en/infraops/installation.asciidoc

```diff
@@ -0,0 +1,110 @@
+
+beta[]
+
+To get up and running with infrastructure monitoring, you need:
+
+* An Elasticsearch cluster and Kibana (version 6.5 or later) with a basic
+license. To learn how to get up and running quickly, see
+{stack-gs}/get-started-elastic-stack.html[Getting started with the
+{stack}].
++
+[TIP]
+==============
+You can skip having to install {es} and {kib} by using our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {es} Service] on
+Elastic Cloud. The {es} Service is available on both AWS and GCP.
+https://www.elastic.co/cloud/elasticsearch-service/signup[Try out the {es}
+Service for free].
+==============
+
+* {beats} shippers installed on each system you want to monitor
+
+[float]
+[[install-beats-for-infra-UI]]
+=== Install {beats} shippers
+
+To populate the <<infrastructure-ui-overview,{infra-ui} UI>> with metrics and
+log data, you need to install and configure the following shippers:
+
+* https://www.elastic.co/products/beats/metricbeat[{metricbeat}] for metrics
+* https://www.elastic.co/products/beats/filebeat[{filebeat}] for log data
+
+To learn how to install and configure the shippers, do one of the following:
+
+* Follow the instructions in the Add Data section of the {kib} home page. Click
+*Add log data* or *Add metrics*, and follow the links for the types of data you
+want to collect.
++
+[role="screenshot"]
+image::add-data.png[]
+
+* If your data source isn't in the list, or you want to install {beats} the old
+fashioned way:
+
+** Follow the 
+{metricbeat-ref}/metricbeat-getting-started.html[{metricbeat} getting started]
+and enable modules for the metrics you want to collect.
+
+** Follow the
+{filebeat-ref}/filebeat-modules-quickstart.html[{filebeat} modules quick start]
+and enable modules for the logs you want to collect. If there is no module
+for the logs you want to collect, see the
+{filebeat-ref}/filebeat-getting-started.html[{filebeat} getting started] to
+learn how to configure inputs.
+
+For either approach, you need to enable modules in {filebeat} and {metricbeat}
+to populate the {infra-ui} UI with data. 
+
+To collect metrics, enable:
+
+* {metricbeat-ref}/metricbeat-module-system.html[{metricbeat} `system` module]
+* {metricbeat-ref}/metricbeat-module-docker.html[{metricbeat} `docker` module]
+* {metricbeat-ref}/metricbeat-module-kubernetes.html[{metricbeat} `kubernetes` module]
+
+To collect logs, enable: 
+
+* {filebeat-ref}/filebeat-module-system.html[{filebeat} `system` module]
+* Any other {filebeat-ref}/filebeat-modules.html[{filebeat} modules] needed for
+your environment, such as `apache2`, `redis`, and so on
+* The {filebeat-ref}/filebeat-input-docker.html[docker input in {filebeat}]
+
+
+// REVIEWERS: This language is too wishy washy. Can you help me come up with
+// better language to help users understand when this is required?
+
+You may also need to add processors to your configuration to collect metadata
+about containers or hosts where your services are running:
```

Comment:
This is how I understand it:

* If the user enables the filebeat `docker` input to tail docker logs or the metricbeat docker module, she should also enable the `add_docker_metadata` processor.
* If the user enables the filebeat `docker` input to tail kubernetes logs or the metricbeat kubernetes module, she should also enable the `add_kubernetes_metadata` processor.
* The `add_cloud_metadata` and `add_host_metadata` processors are enabled by default starting with 6.5.

We could structure this section to match the three node types currently supported, e.g.

* If the user wants to collect host system logs and see hosts and their metrics in the Infrastructure UI, she should enable the `system` filebeat and metricbeat modules.
* If the user wants to collect docker container logs and see containers and their metrics in the Infrastructure UI, she should enable the `docker` filebeat input and metricbeat module as well as the `add_docker_metadata`. Autodiscovery can help to automate that.
* If the user wants to collect kubernetes container logs and see pods and their metrics in the Infrastructure UI, she should enable the `docker` filebeat input and `kubernetes` metricbeat module as well as the `add_kubernetes_metadata`. Autodiscovery can help to automate that.